feature_tests/common/constants.py

```diff
@@ -59,6 +60,9 @@
     DocumentPointer: DOCUMENT_POINTER_TABLE_DEFINITION,
 }
 
+PERMISSIONS_BUCKET = "permissions_bucket"  # This should match the value in TF outputs
```

Comment:
Is this value right or does it get changed later? its name is now 

"${local.prefix}--authorization-store"

Reply:
oh that comment's old actually, now it's just a dummy name for the local run (i.e. when mocking s3)

Reply:
so i'll rm the comment